webapp/sources/rudder/rudder-core/src/main/resources/reportsSchema.sql

```diff
@@ -456,7 +456,7 @@ CREATE TABLE NodeFacts (
 );
 
 
-CREATE TYPE score AS enum ('A', 'B', 'C', 'D', 'E');
+CREATE TYPE score AS enum ('A', 'B', 'C', 'D', 'E', 'F', 'X');
```

Comment:
This leads to SQL errors for people with the beta2 since there is no migration. Pondering if it's important, not sure, but we should at least add in release notes: 
"if you upgrade from a beta version, you need to do: 
```
rudder=> ALTER TYPE score ADD VALUE 'F';
ALTER TYPE
rudder=> ALTER TYPE score ADD VALUE 'X';
ALTER TYPE
rudder=>
```

Reply:
We don't support update between beta, we never did nor we want to support it